ports/re8/README.md

```diff
@@ -0,0 +1,14 @@
+## Notes
+
+Thanks to [OcO](https://oco.itch.io/) for making this wonderful demake 
```

Comment:
Suggestion to have the link point to the actual game's page

Reply:
Fixed